docs/source/backends/openwrt.rst

```diff
@@ -595,6 +632,14 @@ Will be rendered as follows::
             option ifname 'eth0.2'
             option proto 'none'
 
+    config device 'device_lan_bridge'
+            option igmp_snooping '1'
+            option name 'lan_bridge'
```

Comment:
the name should be `br-lan_bridge`. Let's simplify the example and call it just lan, so it will be called `br-lan`.